spec/requests/affiliates_spec.rb

```diff
@@ -212,10 +212,11 @@
 
       expect(page).to have_link("Edit")
       expect(page).to have_button("Delete")
+      click_on "Close"
```

Comment:
Since we now have an overlay, Capybara throws an "element click intercepted" error without this.